### PR TITLE
Updated docs for Source Controller v0.4.1

### PR DESCRIPTION
**Pull request by rashedkvm**
Signed-off-by: Rashed Kamal <krashed@vmware.com>

Which other branches should this be merged with (if any)?

N/A

### CODE REVIEW

File: source-controller/install-source-controller.md [hunk 1]
```diff
@@ -46,27 +47,39 @@ To install Source Controller:
...
+    aws_iam_role_arn           string  Optional: The AWS IAM Role ARN to attach to the source controller service account  
+    ca_cert_data               string  Optional: PEM Encoded certificate data for image registries with private CA.
     ```
 
-3. (Optional) Enable Source Controller to connect to image registries that use self-signed or private certificate authorities.
-If a certificate error `x509: certificate signed by unknown authority` occurs, this option can be used to trust additional certificate authorities.
+3. (Optional) There are two optional fields that can override Source Controller's default installation setting.
 
-    To provide a custom certificate, create a file named `source-controller-values.yaml` that includes the PEM-encoded CA certificate data.
+- `ca_cert_data` Enables Source Controller to connect to image registries that use self-signed or private certificate authorities.If a certificate error `x509: certificate signed by unknown authority` occurs, this option can be used to trust additional certificate authorities.
```
Comment (heyjcollins) [inline source-controller/install-source-controller.md:60]: Need to add a space character in this section: 
`private certificate authorities.If`
-- SHOULD BE --
`private certificate authorities. If`

File: source-controller/reference.md [hunk 2]
```diff
@@ -21,3 +22,59 @@ repository, exposing the resulting source artifact at a URL defined by `.status.
...
+    url: https://repo1.maven.org/maven2
+  interval: 5m0s
+  timeout: 1m0s
+```
+
+`MavenArtifact` resolves artifact from a Maven repository, exposing the resulting artifact at a URL defined by `.status.artifact.url`.
+
+The `interval` determines how often to check artifact for changes. Setting this value too high will result in delays in discovering new sources, while setting it too low may trigger a repository's rate limits.
+
+Repository credentials may be defined as secrets referenced from the resources at `.spec.repository.secretRef`. Secret referenced by `spec.repository.secretRef` is parsed as following:
```
Comment (heyjcollins) [inline source-controller/reference.md:49]: `parsed as following:`
-- SHOULD BE --
`parsed as follows:`

**Review by danfein — APPROVED** [review]
lgtm, I didn't see anything that Josh didn't already catch.

File: source-controller/install-source-controller.md (hunk 1 above)
Comment (rashedkvm) [reply source-controller/install-source-controller.md:60]: Done

**Comment by rashedkvm** [comment]
@heyjcollins updated PR with requested changes. This PR is ready to merge.

**Comment by snneji** [comment]
Thanks @rashedkvm

**Comment by heyjcollins** [comment]
LGTM. Thanks Rashed!

**Pull request merged**